ratatui-widgets/src/block.rs

```diff
@@ -22,87 +22,194 @@ use crate::borders::{BorderType, Borders};
 
 mod padding;
 
-/// Base widget to be used to display a box border around all other built-in widgets.
+/// A widget that displays borders, titles, and padding around other widgets.
```

Comment:
nit: `displays` sounds off to me in context of padding (but it can be just me)
```suggestion
/// A widget that adds borders, titles, and padding around other widgets.
```